mantis/acquisition/acq_engine.py

```diff
@@ -564,8 +595,11 @@ def update_lf_acquisition_rates(self, lf_exposure_times: list):
             return
 
         # Determine label-free acq timing
-        oryx_framerate = float(self.lf_acq.mmc.getProperty('Oryx', 'Frame Rate'))
+        oryx_framerate = float(
```

Comment:
Unfortunately, there is no standard way to get the camera framerate in MicroManager - this will work for Spinnaker cameras which have a `Frame Rate` property exposed through the device adapter. Other camera manufacturers may call it something different. Still. I do agree with the change you are making here as `self.lf_acq.mmc.getCameraDevice()` is more general than `Oryx`. As a step further, maybe we can make a `microscope_operations.get_camera_framerate()` function which will standardize retrieving the camera framerate across the few different camera manufacturers we use at the Biohub?

Reply:
This is a pretty weak suggestion at this point - we have hardcoded device name at many places throughout the code, we can also clean these up in another PR, I know this is not the point of this PR